test/framework/easyconfigs/toy-0.0-patches.eb

```diff
@@ -0,0 +1,32 @@
+name = 'toy'
```

Comment:
I don't think there's a need to add another toy easyconfig. `toy-0.0.eb` also contains a patch, so use that to test `fetch_patches` with the default `None`, so `self.cfg['patches'] is used

the other cases can be tested by supplying a list of patches to `fetch_patches`